packages/serverpod/lib/src/endpoints/insights.dart

```diff
@@ -177,4 +178,19 @@ class InsightsEndpoint extends Endpoint {
     }
     return await HotReloader.hotReload();
   }
+
+  /// Returns the desired structure of the database.
+  /// See also:
+  /// - [getCurrentDatabaseDefinition]
+  Future<DatabaseDefinition> getTargetDatabaseDefinition(
+      Session session) async {
+    return session.serverpod.serializationManager.getTargetDatabaseDefinition();
+  }
+
+  /// Returns the current structure of the database.
+  /// See also:
+  /// - [getTargetDatabaseDefinition]
+  Future<DatabaseDefinition> getCurrentDatabaseDefinition(Session session) {
```

Comment:
Sorry for being a pita here. Calling this `current` may not be the best naming as the target is also current in a sense. How about `getActualDatabaseDefinition` or there may be some even better naming like `live` or `existing`. What do you think?

Reply:
Kind of like `getLiveDatabaseDefinition`.

Reply:
6c9a15d5